backend/tests/services/readers/test_kb_permission_resolver.py

```
@@ -0,0 +1,325 @@
+        patch.object(db, "query") as mock_query,
+        patch(
+            "app.services.readers.kb_permissions.kbPermissionResolver"
+        ) as mock_resolver,
+        patch(
+            "app.services.share.knowledge_share_service.is_restricted_analyst",
+            return_value=False,
+        ),
+    ):
+        mock_query.return_value.filter.return_value.first.return_value = mock_kb
+
+        has_access, role, is_creator = service.get_user_kb_permission(
```

Ruff (0.15.11)

[warning] Unpacked variable `role` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)